mm2src/coins/siacoin.rs

```diff
@@ -652,12 +684,19 @@ impl MarketCoinOps for SiaCoin {
         Ok(public_key.to_string())
     }
 
-    fn sign_message_hash(&self, _message: &str) -> Option<[u8; 32]> { unimplemented!() }
+    fn sign_message_hash(&self, _message: &str) -> Option<[u8; 32]> {
+        // TODO: Implement message signing for SiaCoin.
+        None
+    }
 
-    fn sign_message(&self, _message: &str) -> SignatureResult<String> { unimplemented!() }
+    fn sign_message(&self, _message: &str) -> SignatureResult<String> {
+        // TODO: Implement message signing for SiaCoin.
+        MmError::err(SignatureError::InternalError("Not implemented".to_string()))
+    }
 
     fn verify_message(&self, _signature: &str, _message: &str, _address: &str) -> VerificationResult<bool> {
```

Comment:
I would prefer to refactor MarkerCoinOps to remove these methods used exclusively by RPC methods.

I believe `sign_message_hash`, `sign_message` and `verify_message` can be moved into something like "OptionalRpcOps" since they are not actually needed for all implementations.

This is not a blocker for this PR, but it would be an appreciated inclusion.